terra_notebook_utils/drs.py

```diff
@@ -23,6 +24,9 @@
 
 DRSInfo = namedtuple("DRSInfo", "credentials bucket_name key name size updated")
 
+class InaccessibleDrsUrlException(Exception):
```

Comment:
Consider renaming to something like `GSBlobInnaccessible`. In a future PR, we might consider raising this error for all drs:// and gs:// URLs that fail to resolve because of non-existence or acces-denied.